src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabServerConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import jakarta.inject.Inject;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.openqa.selenium.By;
+
+public class GitLabServerConfig extends PageAreaImpl {
+
+    @Inject
+    public GitLabServerConfig(Jenkins jenkins) {
+        super(jenkins, "/com-dabsquared-gitlabjenkins-connection-GitLabConnectionConfig/connections");
+    }
+
+    public String configureServer(String url, String token) {
+        control("name").set("servername");
+        control("url").set(url);
+        control(By.cssSelector("button.credentials-add-menu")).click();
```

Comment:
Using cssSelector is usually fragile... are there no other options like `by.id` or `by.path` available?

Also, please use `by` wrapper instead of `By` native. The wrapper contains extra logic tailored to jenkins UI